hugegraph-store/hg-store-node/src/main/java/org/apache/hugegraph/store/node/metrics/JRaftMetrics.java

```diff
@@ -172,35 +168,33 @@ private static void registerHistogram(String group, String name,
 
         String baseName = PREFIX + "." + name.toLowerCase();
 
-        HistogramWrapper wrapper = toWrapper(histogram);
-
-        Gauge.builder(baseName + ".median", wrapper, (d) -> d.getSnapshot().getMedian())
+        Gauge.builder(baseName + ".median", histogram, (d) -> d.getSnapshot().getMedian())
              .tags(tags).register(registry);
-        Gauge.builder(baseName + ".min", wrapper, (d) -> d.getSnapshot().getMin())
+        Gauge.builder(baseName + ".min", histogram, (d) -> d.getSnapshot().getMin())
```

Comment:
we can also rename "(d) ->" to "h ->"

Reply:
Thanks for your suggestion, I will optimize it